### PR TITLE
Resolving, some but not all, GEN-ART comments.

### PR DESCRIPTION
**Pull request by seanturner**
This deals with some of Meral's GEN-ART comments.

### CODE REVIEW

File: rtcweb-overview.xml [hunk 1]
```diff
@@ -431,8 +431,8 @@
 
       <t>If the two Web servers are operated by different entities, the
       inter-server signaling mechanism needs to be agreed upon, either by
-      standardization or by other means of agreement. Existing protocols (for
-      example SIP <xref target="RFC3261"/> or XMPP <xref target="RFC6120"/>)
+      standardization or by other means of agreement. Existing protocols
+      (e.g. SIP <xref target="RFC3261"/> or XMPP <xref target="RFC6120"/>)
```
Comment (alvestrand) [inline rtcweb-overview.xml:435]: I have a principled objection to using Latin-based abbreviations when there are common English words that mean exactly the same thing. It provides a stumbling block for international readers. (I had to look up "e. g." to make sure there was no difference in meaning - which there isn't, according to Wiktionary).

**Comment by seanturner** [comment]
I can see putting it back, but we should be consistent there were two e.g. and two for examples.  My matchy-matchy tendencies getting to me.

**Comment by alvestrand** [comment]
:-) I can flip it in the other direction. Sometimes I show off my English skills, othertimes I want to write simple English.
Trapped by my own filters .... the review ended up in my gen-art inbox, which I stopped reading years ago when I left the gen-art team....

**Comment by seanturner** [comment]
No worries.  I would go with whatever the RFC editor is going to end up doing, but to be honest I don’t know which they’d prefer.

spt

**Pull request merged**